api/proto/job/job.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+option go_package = "teleport-job-worker/api/proto/job";
+package job;
+
+message StartRequest {
+  job.Command command = 1;
+  job.Resources resource_limits = 2;
+}
+
+message StopRequest {
+  string id = 1;
+}
+
+message QueryRequest {
+  string id = 1;
+}
+
+message OutputRequest {
+  string id = 1;
+}
+
+message Resources {
+  int32 memory = 1;
```

Comment:
Would it make sense to give these names that better indicate how they apply controls?

Could you add a comment documenting each field?

Reply:
Yeah, I think that's a good idea. Done.